packages/web/__tests__/test-utils.tsx

```diff
@@ -8,29 +8,37 @@ import { ReactNode } from "react";
 // eslint-disable-next-line import/no-extraneous-dependencies
 import SuperJSON from "superjson";
 
+import { WalletSelectProvider } from "~/hooks";
 import { MultiLanguageProvider } from "~/hooks/language/context";
 import { AvailableFlags } from "~/hooks/use-feature-flags";
 import { AppRouter } from "~/server/api/root-router";
+import { StoreProvider } from "~/stores";
 
 export const trpcReact = createTRPCReact<AppRouter>();
 
 const queryClient = new QueryClient();
 export const withTRPC = ({ children }: { children: ReactNode }) => (
-  <MultiLanguageProvider defaultLanguage="en">
-    <trpcReact.Provider
-      client={trpcReact.createClient({
-        transformer: SuperJSON,
-        links: [
-          httpBatchLink({
-            url: "http://localhost:3000/trpc",
-          }),
-        ],
-      })}
-      queryClient={queryClient}
-    >
-      <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
-    </trpcReact.Provider>
-  </MultiLanguageProvider>
+  <StoreProvider>
```

Comment:
Would be nice to see this being more configurable in the future.

Reply:
Certainly. It's designed to be flexible enough to extend as issues arise during testing